index.js

```diff
@@ -1,88 +1,239 @@
+              groups: [
+                'builtin',
+                'subpath',
+                'aliased-external',
+                'external',
+                'internal',
+                'aliased-internal',
+                'parent-composables',
+                'parent-components',
+                'parent',
+                'sibling',
+                'index'
+              ],
+              // The default is `['^~/.+', '^@/.+']`
+              // But we want to group `~*/** */` before other external imports,
+              // and `@/**/*` after normal internal imports.
+              // So we need to reset it here.
+              internalPattern: [],
 
-    '@typescript-eslint/brace-style': ['error', '1tbs', {
-      allowSingleLine: true
-    }],
-    '@typescript-eslint/comma-dangle': ['error', 'never'],
-    '@typescript-eslint/consistent-type-definitions': 'off',
-    '@typescript-eslint/consistent-type-imports': ['error', {
-      prefer: 'type-imports',
-      disallowTypeAnnotations: false,
-      fixStyle: 'inline-type-imports'
-    }],
-    '@typescript-eslint/no-empty-interface': 'off',
-    '@typescript-eslint/no-unused-vars': ['error', {
-      argsIgnorePattern: '^_',
-      destructuredArrayIgnorePattern: '^_'
-    }],
-    '@typescript-eslint/no-use-before-define': 'off',
+              customGroups: [
+                {
+                  groupName: 'aliased-external',
+                  elementNamePattern: '~.*/.*'
+                },
+                {
+                  groupName: 'parent-components',
+                  elementNamePattern: '@/components/.*'
+                },
+                {
+                  groupName: 'parent-composables',
+                  elementNamePattern: '@/composables/.*'
+                },
+                {
+                  groupName: 'aliased-internal',
+                  elementNamePattern: '@/.*'
+                }
+              ]
+            }
+          ],
 
-    'vue/attributes-order': 'off',
-    'vue/comma-dangle': ['error', 'never'],
-    'vue/component-name-in-template-casing': ['warn', 'PascalCase'],
-    'vue/custom-event-name-casing': ['error', 'kebab-case'],
-    'vue/define-macros-order': 'off',
-    'vue/html-closing-bracket-newline': 'off',
-    'vue/no-extra-parens': 'off',
-    'vue/no-useless-v-bind': ['error', { ignoreStringEscape: true }],
-    'vue/no-v-text-v-html-on-component': 'off',
-    'vue/singleline-html-element-content-newline': 'off'
-  }
+          'perfectionist/sort-named-imports': ['error', {
+            type: 'custom',
+            order: 'asc',
+            ignoreCase: false
+          }]
+        }
+      })
+      .override('antfu/vue/rules', {
+        rules: {
+          'vue/attributes-order': 'off',
+          'vue/comma-dangle': ['error', 'never'],
+          'vue/component-name-in-template-casing': ['warn', 'PascalCase'],
+          'vue/custom-event-name-casing': ['error', 'kebab-case'],
+          'vue/define-macros-order': 'off',
```

Comment:
(line 163) can be enabled but the default is defineOptions on top, I don't think I like that though we rarely use defineOptions. Also, the default doesn't have defineModel.

Reply:
Then let's leave it off for now?

Reply:
yeah I'm fine. I think we already are pretty consistent across the codebases and usually follow props -> events -> models -> ... other stuff ... -> options at bottom